backend/main.cpp

```diff
@@ -95,7 +104,7 @@ void handle_request(const std::string &request, ip::tcp::socket &frontend_socket
             send_method_not_allowed_response(frontend_socket);
             return;
         }
-
+        const auto timer = std::chrono::high_resolution_clock::now();
```

Comment:
I think it's worth thinking about whether we should include the json parsing, payload generating, etc. in the timing, or just purely time the FPGA processing time. Perhaps we could do both? I'm okay with this for now, since those other things should take very little time anyways...